src/lsm/manifest_log_fuzz.zig

```diff
@@ -37,12 +36,6 @@ const manifest_log_options = ManifestLogOptions{
     .forest_table_count_max = schema.ManifestNode.entry_count_max * 100,
 };
 
-const pace = @import("manifest_log.zig").Pace.init(.{
-    .tree_count = manifest_log_options.forest_tree_count(),
-    .tables_max = manifest_log_options.forest_table_count_max,
-    .half_bar_compact_blocks_extra = constants.lsm_manifest_compact_blocks_extra,
-});
-
```

Comment:
_should_ we be using this?

Reply:
I will look into adding it to the fuzzer, but feel free to remove it for now since it is dead code!